jans-cedarling/bindings/cedarling_c/benchmarks/benchmark_cedarling.c

```
@@ -3,287 +3,190 @@
-        MULTI_ISSUER_CONFIG,
-        sizeof(MULTI_ISSUER_CONFIG),
-        "{"
-        "\"CEDARLING_APPLICATION_NAME\":\"BenchmarkApp\","
-        "\"CEDARLING_JWT_SIG_VALIDATION\":\"disabled\","
-        "\"CEDARLING_JWT_STATUS_VALIDATION\":\"disabled\","
-        "\"CEDARLING_LOG_TYPE\":\"off\","
-        "\"CEDARLING_JWT_SIGNATURE_ALGORITHMS_SUPPORTED\":[\"HS256\"],"
-        "\"CEDARLING_POLICY_STORE_LOCAL_FN\":\"%s\""
-        "}",
-        multi_issuer_policy
+static void emit_skipped(const char *id, const char *reason) {
```

Clang (14.0.6)

[warning] parameter name 'id' is too short, expected at least 3 characters

(readability-identifier-length)